editions/tw5.com/tiddlers/filters/Filter Operators.tid

```diff
@@ -21,6 +21,8 @@ A <<.def "filter operator">> is a predefined keyword attached to an individual [
 
 ''Important:'' Each first [[step|Filter Step]] of a [[filter run|Filter Run]] not given any input titles receives the output of <$link to="all Operator">[all[tiddlers]]</$link> as its input.
 
+[[Filters]] and Filter Operators are most commonly used in the ListWidget, SetWidget and [[Filtered Transclusions|Transclusion in WikiText]] `{{{ filter }}}`
```

Comment:
@AnthonyMuscio ... I think this PR is obsolete now. It probably wasn't as you created it. 

The `[[Filters]]` link is a duplicate of the `[[filter|Filters]]` link in line 20 ... 
`[[Filtered Transclusions|Transclusion in WikiText]]` .. Is linked at the [Filters](https://tiddlywiki.com/#Filters) tiddler. 
Lists are also mentioned in the Filters tiddler. 
------
@AnthonyMuscio 
IMO this PR can be closed, because it only contains duplicated info now.